plugins/inputs/interrupts/README.md

```diff
@@ -8,6 +8,10 @@ The interrupts plugin gathers metrics about IRQs from `/proc/interrupts` and `/p
   ## To filter which IRQs to collect, make use of tagpass / tagdrop, i.e.
   # [inputs.interrupts.tagdrop]
     # irq = [ "NET_RX", "TASKLET" ]
+  #
+  # To report cpus as tags instead of fields use cpus_as_tags
+  # cpus_as_tags = false
```

Comment:
Can you tweak this to `cpu_as_tag` (will match better with unbound plugin's `thread_as_tag`)?